natalie/cards-template1b.js

```diff
@@ -0,0 +1,122 @@
+// Error-detecting version
+
```

Comment:
Looking good, but I've found one thing you'll want to look into. name(1) produces the result of Ace of Diamond, but color(1) produces "black". This should be returning red. Take another look at your color function and see if you can tell what the error is, and then check it against the homework solutions.